src/pages/Main/Main.js

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from 'react';
+import Slide from './Slide/Slide';
+import ProductCard from '../../components/ProductCard/ProductCard';
+import './main.scss';
+
+function Main() {
+  const [imgId, setImgId] = useState(1);
+  const [values, setValues] = useState({
+    imgURL: '',
+    korTitle: '',
+    engTitle: '',
+    details: '',
+    roastedDate: '',
+    price: '',
+  });
+  const [newValue, setNewValue] = useState();
+
+  // mock data 가져오기
+
+  async function request() {
+    const res = await fetch('http://localhost:3000/data/productCardBest.json');
```

Comment:
```suggestion
    const res = await fetch('/data/productCardBest.json');
```
mock data로 fetch 할 때는 localhost:3000 부분은 생략 가능합니다!